tests/tests_data/basic_scenes.py

```diff
@@ -0,0 +1,31 @@
+from manim import *
+
+#This module is used for testing purpose only
+
+class SquareToCircle(Scene):
+    def construct(self):
+        circle = Circle()
+        square = Square()
+        square.flip(RIGHT)
+        square.rotate(-3 * TAU / 8)
+        circle.set_fill(PINK, opacity=0.5)
+
+        self.play(ShowCreation(square))
+        self.play(Transform(square, circle))
+        self.play(FadeOut(square))
+
+class WriteStuff(Scene): 
+    def construct(self): 
+        example_text = TextMobject(
+            "This is a some text",
+            tex_to_color_map={"text": YELLOW}
+        )
+        example_tex = TexMobject(
+            "\\sum_{k=1}^\\infty {1 \\over k^2} = {\\pi^2 \\over 6}",
+        )
+        group = VGroup(example_text, example_tex)
+        group.arrange(DOWN)
+        group.set_width(FRAME_WIDTH - 2 * LARGE_BUFF)
+
+        self.play(Write(example_text))
+        self.play(Write(example_tex))
```

Comment:
Are these scenes being test at all? I don't see a `SceneTester` instance.

Reply:
Yes, I, fact I wanted to delete this file since it is now useless.

What do you think ?

Reply:
Yeah this can go.

Reply:
My mistake. This file is used for the CLI test in `test_CLI.py` 
i will precise this in the comment.